src/main/java/lotto/domain/Lotto.java

```diff
@@ -1,24 +1,25 @@
 package lotto.domain;
 
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 public class Lotto {
 
-    private List<Integer> lotto;
-
-    public Lotto(List<Integer> lotto) {
+    private List<LottoNumber> lotto;
 
+    public Lotto(List<LottoNumber> lotto) {
+        Collections.sort(lotto);
         this.lotto = lotto;
     }
 
-    public List<Integer> getLotto(){
-
-        return lotto;
+    public final List<LottoNumber> getLotto(){
+        return new ArrayList<>(lotto);
     }
 
     public int checkSameCount(Lotto userLotto){
         int count = 0;
-        for (Integer number : userLotto.getLotto()) {
+        for (LottoNumber number : userLotto.lotto) {
             count += lotto.contains(number) ? 1 : 0;
         }
```

Comment:
이 부분도 stream을 이용하여 간단히 처리할 수도 있습니다
```suggestion
        count = (int)userLotto.lotto.stream()
                .filter(lotto::contains)
                .count();
```